### PR TITLE
Updated German Translation

### PR DESCRIPTION
**Pull request by elloellie**
i'm not a native speaker of german, but i'm minoring in it in college. me and my friends wanted to play a modpack in german to practice, so i decided to translate some mods! i noticed the farmer's delight translation wasn't currently up to date and covered what needed it!

anyways, this might not be 100% correct since i'm not a native speaker. if anything's incorrect anyone can feel free to edit it, or let me know and i'll spruce it up.

### CODE REVIEW

**Comment by ConductiveFoam** [comment]
First things first, thank you! This is solid, especially for a non-native speaker, but there's some peculiarities.

I've never heard "Küchentheke" as a word, really. When I hear "Theke" I think of a bar or restaurant, where someone serves others food & drink. I think a more idiomatic word is "Küchenschrank". It might be a regional thing though, I'm fairly south.

Why did you choose "*Küchen*brett"? "Schneidbrett" seems more common to me.

"Nudeln mit Fleischfrikadellen" should probably be "Nudeln mit Fleischbällchen", "Fleischfrikadellen" is kinda doppelt gemoppelt, as we say.

I think it'd be better to call it "Ein Stück honigglasierter Schinken", "Teller von" is ambiguous to me in how much it is — could be a portion, could be the whole meal.

There's a typo in the `when_feeding` texts: "zähmten" should be "gezähmten", although the text itself feels a bit off to me. I think it should be "Verleiht, wenn es an gezähmte [X] verfüttert wird:".

Typo in "Küchenmessel" on L166, and it's not all animals, or *more* meat, so possibly "..., um Schinken von Schweinen zu bekommen." Although that doesn't mention Hoglins — which might actually be a good thing, so people can figure it out themselves.

"Futterpflanze" is backwards, it means feed. "Pflanzenfutter" was fine.

"Belasse Organischen Kompost" sounds arcaic tbh, and the Teilsatz is grammatically weird; the "der" should be "ihn", and there should be a "zu" in front of "lassen". The construction itself isn't something we'd say in German, though, about compost changing into soil. Don't really know how to describe it, sorry. The previous translation was fine imo.

The word order of the Mushroom Colony advancement description is off, we'd say "Pflanze Pilze in Dunkelheit auf reichhaltiger Erde, dass sie eine Pilzkolonie formen. Nutze eine Schere, um sie zu ernten."

The verb in the netherite upgrade advancement should be "verbessern" imo, "machen" implies creating from scratch to me.

"Apotheke-o-lade" is a fantastic portmanteau imo.
The text should read "Trink eine Flasche heißer Schokolade, um einen gefährlichen Effekt abzuschütteln."

Typos:
L24 "Rote-Be*e*te-Kiste"
L37 "Wildzwiebel" instead of "Wildezwiebel" — if it was two words, the e at the end would be correct, in a composite it's dropped though
L88 "Melone*n*stieleis"
L116 "Gebackener Kabeljaueintopf", this works perfectly fine as a composite
L130 "Eine Schüssel gefüllter Kürbis", although "Eine Portion" might be better still
L164 "Baue alles Obst, Gemüse, [...]", and there's a bit of the English text trailing that should be removed
L168 "Abernte" and "Pflanzen", however we would say "Ernte krautige Pflanzen [...] Küchenmesser ab, [...]"

And one thing that isn't on you, but while you're at it I think you should change "Master Chef" to "Chefkoch", that's a common enough word.

I totally feel your pain, German composites are really hard. Fun to play with once you figure them out, though

**Comment by elloellie** [comment]
ah thank you! yeah i am AWFUL about typos lol. i tried my best to catch them but i'll admit that some of these were done on very little sleep :-p

i'll go through and update it with your feedback! i think i maaaay have been translating from an old file, since i tried to only translate things that weren't covered. thank you so much!! and yeah, we haven't covered a *ton* on composites. this was one of the first mods i translated and i was perhaps a bit overeager with throwing words together.

most of the word choices came either from whatever i knew or dict.cc as a back-up. i've noticed a lot of their translations tend to be clunkier than other english to german dictionaries, so i think that may have something to do with some of the weirder constructions lmao.

but again thank you!! i really appreciate you going through it like that :-)

edit: i see what i did with the already translated things that i re-translated! i was comparing the outdated german translation to the current english one. i think the layout of it changed slightly at some point, so i overlooked some that were already done. oops!

**Comment by vectorwing** [comment]
Thanks for updating the lang! And thank you Foam, for giving it a proper review.

Usually, I'd go through a lang outside my comprehension by using Google Translate to get a feel for what's being translated to, but having a native speaker go through it is a lot cleaner. Please let me know when the review is over, and I'll merge it! 👍

**Comment by elloellie** [comment]
i went through and added/changed everything foam commented on! i think it should be good to go, but feel free to wait for foam to give it one more once over!

File: src/main/resources/assets/farmersdelight/lang/de_de.json [hunk 1]
```diff
@@ -145,14 +145,14 @@
...
+  "farmersdelight.container.pantry": "Holzküchenschrank",
 
   "farmersdelight.tooltip.cooking_pot.empty": "Leer",
   "farmersdelight.tooltip.cooking_pot.single_serving": "Hält 1 Portion:",
   "farmersdelight.tooltip.cooking_pot.many_servings": "Hält %s Portionen:",
 
-  "farmersdelight.tooltip.dog_food.when_feeding": "Effekte, wenn einem zähmten Wolf zu fressen gegeben werden:",
-  "farmersdelight.tooltip.horse_feed.when_feeding": "Effekte, wenn einem zähmten Reittier zu fressen gegeben werden:",
+  "farmersdelight.tooltip.dog_food.when_feeding": "Verleiht, wenn es an gezähmte Wolf verfüttert wird:",
+  "farmersdelight.tooltip.horse_feed.when_feeding": "Verleiht, wenn es an gezähmtes Pferd verfüttert wird:",
```
Comment (ConductiveFoam) [inline src/main/resources/assets/farmersdelight/lang/de_de.json]: "gezähmte Wolf" and "gezähmtes Pferde" are wrong, unfortunately. You didn't make the cases and number agree. It should be "gezähmte Wölfe" and "gezähmte Pferde". "gezähmte" in this context is a plural accusative participle.
If you want this to be singular, it needs an indefinite article. Works too, whether singular or plural is a stylistic choice at that point

File: src/main/resources/assets/farmersdelight/lang/de_de.json [hunk 2]
```diff
@@ -161,15 +161,15 @@
   "farmersdelight.advancement.plant_rice": "Wurzeln eintauchen",
   "farmersdelight.advancement.plant_rice.desc": "Pflanze Reis in seichtes Wasser",
   "farmersdelight.advancement.plant_all_crops": "Fruchtfolge",
-  "farmersdelight.advancement.plant_all_crops.desc": "Baue alle Obst, Gemüse, Pilz und Wurzel an, dass du finden kannst.  every fruit, vegetable, fungus or root you can find!",
+  "farmersdelight.advancement.plant_all_crops.desc": "Baue alles Obst, Gemüse, Gemüse, Pilz und Wurzel an, dass du finden kannst!",
```
Comment (ConductiveFoam) [inline src/main/resources/assets/farmersdelight/lang/de_de.json]: Now "Gemüse" is there twice

**Review by ConductiveFoam — COMMENTED** [review]
Looked over your changes. There's one grammatical thing in there, which I totally understand — everyone makes these case mistakes sometimes.

Thank you!

**Comment by elloellie** [comment]
thank you for going over it again! i have prettyyyyyy bad adhd so i almost always need someone to go back through my work for small and dumb mistakes like that lol. okay! that should be it @vectorwing i think we're good to go! :-) thank you foam for your help!!!

EDIT: i also forgot to mention! i changed "rope" from "Seil" to "Strohseil" (straw rope). i did this because *so* many other mods added rope of some kind or another. that was actually a personal tweak for my modpack, since i wanted both this mod's and druidcraft's rope, but if you'd like me to change it back to just "Seil" i can totally do that!

**Comment by ConductiveFoam** [comment]
Looks good now, thanks a bunch! :)

I see why you did that change, didn't even really see it. Makes sense to me, a bunch of different rope items irk me a bit too.

**Comment by vectorwing** [comment]
> i changed "rope" from "Seil" to "Strohseil" (straw rope)

That's OK! Deliberations between languages are fine if it helps readability.
Since the review seems to be over, I'll merge it now. You can make a new PR anytime to propose more updates. 👍

**Pull request merged**